.kokoro/runintegrationtests.sh

```diff
@@ -2,11 +2,15 @@
 
 set -e
 
-export GOOGLE_APPLICATION_CREDENTIALS="$KOKORO_KEYSTORE_DIR/73609_cloud-sharp-jenkins-compute-service-account"
-export REQUESTER_PAYS_CREDENTIALS="$KOKORO_KEYSTORE_DIR/73609_gcloud-devel-service-account"
+SCRIPT=$(readlink -f "$0")
+ROOT_DIR=$(dirname "$SCRIPT")
```

Comment:
I think "ROOT_DIR" is a misleading name here. It sounds like it's the root of the repository, when it's actually just the directory containing the script. How about SCRIPT_DIR?

Reply:
Done